package.json

```diff
@@ -44,8 +44,11 @@
     ]
   },
   "dependencies": {
+    "i18next": "^23.10.1",
+    "i18next-browser-languagedetector": "^7.2.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-i18next": "^14.1.0"
```

Comment:
Are all of these required for the application to run in production or can some go to `devDependencies`?

Reply:
They can be moved! Thanks!